calico_containers/calico_ctl/node.py

```diff
@@ -122,6 +126,10 @@ def validate_arguments(arguments):
     if arguments.get("<DETACH>") or arguments.get("--detach"):
         detach_ok = arguments.get("--detach") in ["true", "false"]
 
+    detach_libnetwork_ok = (not detach_ok or
```

Comment:
But you're checking it on the next line. I don't see the point in checking it twice.